README.md

```diff
@@ -2,3 +2,20 @@
 
 A buildpack for including the ICU library into a Heroku dyno. This is required
 as a dependency for several tools.
+
+## How it works
+
+This buildpack makes ICU 4.8 available in the `/app/icu` directory for both
+compilation and runtime uses. The primary purpose of this is to allow Charlock
```

Comment:
We should definitely link to the Charlock Holmes GitHub repo here. Maybe we should explicitly call out that we are vendoring an old version of ICU specifically for Charlock?